src/src/CustomBuild/define_plugin_sets.h

```diff
@@ -1855,6 +1855,35 @@ To create/register a plugin, you have to :
     #endif
 #endif
 
+#ifndef FEATURE_DOMOTICZ_NO_HTTP
+  #define FEATURE_DOMOTICZ_NO_HTTP  0
+#endif
+
+#if FEATURE_DOMOTICZ_NO_HTTP  // Move Domoticz enabling logic together, without HTTP controllers
```

Comment:
Not really content with the naming of `FEATURE_DOMOTICZ_NO_HTTP`
Maybe better to use something like `FEATURE_NO_HTTP_CLIENT`

Reply:
It disables all HTTP related features in the DOMOTICZ builds, it covers what it does, I guess.

Reply:
It disables more non-Domoticz related stuff, so the name is a bit confusing.

Reply:
Renamed to `FEATURE_NO_HTTP_CLIENT`